### ISSUE TITLE
Update ROS release artifact from 3.1.5-2

### ISSUE THREAD

**Issue by 130s** [start-jsk/openhrp3#22]
To apply [this commit](https://github.com/start-jsk/openhrp3/pull/21) to ROS `.deb`.

---

Just wanted to raise an attention: To release .deb by only incrementing build version (`MAJOR.MINOR.PATCH-BUILD`) [is NOT allowed](http://answers.ros.org/question/76150/when-to-release-build-increment/?answer=76162#post-id-76162) in ROS. So this must not happen often.

I share how to do this JUST IN CASE it's needed in the future by someone else.
1. Update `release tag` on your upstream repository. Do not use automation tools from ROS (eg. `catkin_prepare_release`, `catkin_generate_changelogs`). If the same tag already exists, you might want to remove it first and then push again with your new data that you're releasing this time.
2. Run `bloom-release`. Process normally, push the artifact that's built by bloom to your release repository. Open a pull request (`bloom` will ask you to do so) to modify `rosdistro` with the new `build version`.
3. Go to the web page of the pull request you just opened, and add explanation why you have to increment `build version`. Since `rosdistro` maintainers are very careful with every detail, you have to give sufficient reasoning.

**Comment by 130s** [comment]
Done in rosdistro.

**Comment by k-okada** [comment]
sorry, we need to check this patch --> https://github.com/start-jsk/openhrp3/pull/23

**Comment by 130s** [comment]
Pre-release testing from "source" (not from `release repo`, which means this should be presumable grabbing the latest source) failed, at downstream build test of `hrpsys`.

http://jenkins.ros.org/job/prerelease-hydro-openhrp3/ARCH_PARAM=amd64,UBUNTU_PARAM=precise,label=prerelease/2/consoleFull

```
Check if /tmp/test_repositories/build_repository/devel/share/openhrp3/share/OpenHRP-3.1/idl/OpenHRP/ClockGenerator.idl exists
CMake Error at idl/CMakeLists.txt:120 (message):
  ERROR:
  /tmp/test_repositories/build_repository/devel/share/openhrp3/share/OpenHRP-3.1/idl/OpenHRP/ClockGenerator.idl
  is not present


-- Configuring incomplete, errors occurred!
make: *** [installed] Error 1
CMake Error at hrpsys/catkin.cmake:25 (message):
  Compile hrpsys failed
Call Stack (most recent call first):
  hrpsys/CMakeLists.txt:2 (include)
```

But looks like `ClockGenerator.idl` is installed.

```
-- Installing: /tmp/test_repositories/build_repository/devel/share/OpenHRP-3.1/idl/OpenHRP/ClockGenerator.idl
```

So at this point we _could_ make a judgement call to re-release `OpenHRP3` package (by incrementing build version again). 
@k-okada if you could confirm that there won't be update, I'll release this again. If there are still work ongoing I'll just wait.

**Comment by k-okada** [comment]
On my local environment, it seems ok

**Comment by 130s** [comment]
Ok then I'll increment now

**Comment by 130s** [comment]
Done. We'll see how this goes...

**Comment by 130s** [comment]
Needs redone maybe because of https://github.com/start-jsk/openrtm_aist_core/issues/23

**Comment by 130s** [comment]
Once [`openrtm_aist_core` builds on Groovy](https://github.com/ros/rosdistro/pull/3411), I'll re-release `OpenHRP3` into Groovy, and Hydro.

**Comment by 130s** [comment]
OpenHRP3 [builds fine](http://jenkins.ros.org/view/GbinP64/job/ros-groovy-openhrp3_binarydeb_precise_amd64/42/consoleFull) with the updated `openrtm_aist_core 1.1.0-4`. 

Still, I assume OpenHRP3 needs to be re-released for [this commit](https://github.com/start-jsk/openhrp3/commit/5216c6338a70e14669d4a46273b08d8f8a90c667) to be applied? @k-okada

**Comment by 130s** [comment]
I just re-released OpenHRP3 `3.1.5-5` into Groovy -- it was an old `3.1.5-3` and thus [`hrpsys` didn't build on pre-release](http://jenkins.ros.org/job/prerelease-groovy-hrpsys/ARCH_PARAM=amd64,UBUNTU_PARAM=precise,label=prerelease/3/). Closing.
